.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: ruby
+bundler_args: --without development
+script: "bundle exec rake"
+notifications:
+  email: false
+rvm:
+  - 2.2.0
+  - 2.1.5
+  - 2.1.0
```

Comment:
mo' thorough is mo' better? 

I dunno. Happy to take one of those out if you think it matters.